lib/openxml/elements/paragraph.rb

```diff
@@ -0,0 +1,8 @@
+module OpenXml
+  module Elements
+    class Paragraph < OpenXml::Container
+      namespace :a
```

Comment:
I wasn't sure where to build them. I kept running into tags in the DrawingML that had the same name, but a different namespace. For instance `spur` shows up here in the `p` namespace, but in the drawingML it has a namespace of `pic`. I decided to build what wasn't readily available here for now. I thought we might sort out what belongs where later. I really think that ultimately a completely different gem for each namespace is a good idea.